### PR TITLE
chore(README): updates for beta

### PR DESCRIPTION
**Pull request by russorat**
Updates for beta release.

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
 
-Once we close on the final feature set of what will be in the first release of InfluxDB in the 2.x line, we will move into the beta phase. At that point, our intention is to avoid making breaking changes to the API or the Flux language. However, it still may be necessary to do so. We will do our best to keep this to an absolute minimum and clearly communicate ANY and ALL changes in this regard via the changelog.
+Once we close on the final feature set of what will be in the first GA release of InfluxDB in the 2.x line, we will move into the RC phase. At that point, our intention is to avoid making breaking changes to the API or the Flux language. However, it still may be necessary to do so. We will do our best to keep this to an absolute minimum and clearly communicate ANY and ALL changes in this regard via the changelog.
 
-The beta will still not be recommended for production usage. During the beta period we will focus on bug fixes, performance, and additive features (where time permits).
+The RCs will still not be recommended for production usage. During the RC period we will focus on bug fixes, performance, and additive features (where time permits).
```
Comment (sanderson) [inline README.md]: ~The~ RCs **are not recommended** for production usage. During the RC period, we will focus on... _(Add a comma after "period")_

File: README.md [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
-**Weekly alpha releases with incremental feature additions and changes to the user interface**
+#### Beta
+**Releases every two weeks or as needed**
 
 Planned additions include:
-- Initial alpha release only supports a single user through the UI and the permission assigned via the security token are "full access".  This restriction will be relaxed delivering the ability to define multiple users and change the access permissions provided via the token.
 - Compatibility layer with 1.x including: 1.x HTTP Write API  and HTTP Read API support for InfluxQL
 - Import Bulk Data from 1.x - convert TSM from 1.x to 2.x
-- Delete API w/ predicates for time (and other)
+- Performance tuning, stability improvements, and fine tuning based on community feedback.
```
Comment (sanderson) [inline README.md]: ~fine tuning~ -> fine-tuning

File: README.md [hunk 3]
```diff
@@ -1,36 +1,37 @@
...
+Additional features will arrive during the beta period until we reach general availability (GA). We will be cutting versioned releases at least every two weeks starting in the first release. There will also be nightly builds based off the latest code in master.
 
-Once we close on the final feature set of what will be in the first release of InfluxDB in the 2.x line, we will move into the beta phase. At that point, our intention is to avoid making breaking changes to the API or the Flux language. However, it still may be necessary to do so. We will do our best to keep this to an absolute minimum and clearly communicate ANY and ALL changes in this regard via the changelog.
+Once we close on the final feature set of what will be in the first GA release of InfluxDB in the 2.x line, we will move into the release candidate (RC) phase. At that point, we intend to avoid making breaking changes to the API or the Flux language. However, it still may be necessary to do so. We will do our best to keep this to an absolute minimum and clearly communicate ANY and ALL changes in this regard via the changelog.
```
Comment (pauldix) [inline README.md]: You just trying to give us an out? I would sincerely hope that we make no breaking changes in the RC phase.

File: README.md [hunk 4]
```diff
@@ -1,36 +1,37 @@
...
 
-The beta will still not be recommended for production usage. During the beta period we will focus on bug fixes, performance, and additive features (where time permits).
+RCs are not recommended for production usage. During the RC period, we will focus on bug fixes, performance, and additive features (where time permits).
```
Comment (pauldix) [inline README.md]: I thought people generally suggested using RCs in production. This is the time that we can uncover bugs with real production workloads and the RC phase of a release is when you do that.

File: README.md (hunk 3 above)
Comment (kelseiv) [reply README.md]: how about:
Once we finalize the InfluxDB 2.0 feature set, we’ll deliver InfluxDB 2.0 release candidate (RC). We'll avoid breaking changes in RC unless absolutely necessary and will communicate necessary changes in the changelog.

File: README.md (hunk 4 above)
Comment (kelseiv) [reply README.md]: how about:
We recommend backing up your production environment and testing InfluxDB 2.0 RC in production. During the RC period, we’ll continue to fix bugs and improve performance.

**Pull request merged**